src/breakpoints/index.ts

```diff
@@ -134,13 +134,25 @@ export namespace Breakpoints {
       }
     }
 
+    get isDisposed(): boolean {
+      return this._isDisposed;
+    }
+
+    dispose(): void {
+      if (!this._isDisposed) {
+        this._isDisposed = true;
+        Signal.clearData(this);
+      }
```

Comment:
Maybe we could write it like this to follow the current style (fast return)?

```typescript
    if (this.isDisposed) {
      return;
    }
    this._isDisposed = true;
    Signal.clearData(this);
```

Reply:
Sure!